Sources/StreamChatUI/UIConfig+CurrentUser.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public extension _UIConfig {
+    struct CurrentUser {
+        public var currentUserViewAvatarView: _CurrentChatUserAvatarView<ExtraData>.Type = _CurrentChatUserAvatarView<ExtraData>
+            .self
+        public var avatarView: ChatAvatarView.Type = ChatAvatarView.self
```

Comment:
What is the difference between those two?

Reply:
I will have to investigate